lib/grpc/endpoint.ex

```diff
@@ -95,11 +95,23 @@ defmodule GRPC.Endpoint do
   end
 
   defp parse_servers(servers) do
-    servers
+    case reflection_enabled?() do
+      true -> [{GRPC.Reflection.Service, %{}} | servers]
+      false -> servers
+    end
     |> Enum.map(fn {server, _} -> server end)
     |> List.flatten()
   end
 
+  defp reflection_enabled?() do
+    Application.get_env(:grpc, :reflection_disabled, false) ||
+      convert!(System.get_env("GRPC_REFLECTION_DISABLED"))
+  end
+
+  defp convert!("true"), do: true
+  defp convert!("false"), do: false
+  defp convert!(op) when is_nil(op), do: false
```

Comment:
This kind of conversion is best done close to the config files. However, I don't think we'll move forward with this